File: C++ Programming/Competitive Programming/Coding Challenges 1/program3.cpp
/*
Q3. Accept number of rows and number of columns from user and display below
pattern.

Input : iRow = 5 iCol = 5
Output :$ * * * *
        * $ * * *
        * * $ * *
        * * * $ *
        * * * * $
*/
#include<iostream>
using namespace std;

void Pattern(int iRow,int iCol)
{
    int i=1;
    int j=0;

    while ( i <= iRow )
    {
        j = 1;

        while (j <= iCol)
        {
            if (j-i)
            {
                cout<<"* ";
            }
            else
            {
                cout<<"$ ";
            }
            j++;
        }
        cout<<"\n";
        i++;
    }
    
}

int main()
{
    int iValue1=0,iValue2=0;

    cout<<"Enter The Number of Row: ";
    cin>>iValue1;

    cout<<"Enter The Number of Column: ";
    cin>>iValue2;

    Pattern(iValue1,iValue2);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 6/program3.java
import java.util.Scanner;

class program3 
{
     public static void Pattern(int iRow, int iCol) 
     {
        int i=1;
        int j=0;
        do {
            j=1;
            do {
                if (j - i == 0) 
                {
                    System.out.print("$ ");
                }
                else
                {
                    System.out.print("* ");
                }
                j++;
            } 
            while (j<=iCol);
            System.out.println();
        i++; 
        } 
        while (i<=iRow);  
     }
    public static void main(String[] args) 
    {
        Scanner sobj=new Scanner(System.in);
        int iValue1=0,iValue2=0;

        System.out.print("Enter The Number of Row: ");
        iValue1=sobj.nextInt();

        System.out.print("Enter The Number of Column: ");
        iValue2=sobj.nextInt();

        Pattern(iValue1,iValue2);

        sobj.close();
    }
    
}
